src/components/services.css

```diff
@@ -0,0 +1,17 @@
+
+
+.services{
+  font-family: 'Fjalla One', sans-serif;
+  font-size: 64px;
+}
+
+
+.service-title{
+  font-family: 'Fjalla One', sans-serif;
+  font-size: 32px;
+}
+
+.service-description{
+  font-family: 'Merriweather Sans', sans-serif;
+  font-size: 24px;
+}
```

Comment:
Comment(nb): I don't mind having CSS classes, but with TailwindCSS (TW), the goal is generally to stay away from having many granular CSS classes in separate files. Instead of writing the CSS directly, let's use the TW calsses on the HTML in the components.